### ISSUE TITLE
Getting invalid response status code: 400 in postinstall.js

### ISSUE THREAD

**Issue by KaizenProgrammer** [inngest/inngest#548]
**Describe the bug**
I'm doing `pnpm i -g inngest-cli@latest` and getting the following:

```
.../node_modules/inngest-cli postinstall: Error: Error downloading binary; invalid response status code: 400
.../node_modules/inngest-cli postinstall:     at C:\Users\me\AppData\Local\pnpm\global\5\.pnpm\inngest-cli@0.16.1\node_modules\inngest-cli\postinstall.js:64:23
.../node_modules/inngest-cli postinstall:     at processTicksAndRejections (node:internal/process/task_queues:96:5)
.../node_modules/inngest-cli postinstall: Failed
 ELIFECYCLE  Command failed with exit code 1.
```
 
 
This is after trying `npx inngest-cli@latest dev` with no success.
 
**Expected behavior**
I'm expecting the local dev server to start up. When that didn't work I expected the CLI to install. It appears that the artifact postinstall trying to retrieve is coming back with a bad request for some reason.

**System info**
- inngest-cli version: 0.16.1
- OS: Windows 10 Pro
- Version 22H2  - 19045.3208

**Comment by jpwilliams** [comment]
We're working on Windows builds for the latest versions of our dev server after working through a couple of issues - see the following issues for context.

- #449 
- #521

In the meantime, for Windows we recommend using v0.14 of the dev server. To do this, either:

- If using Node < 18.16.0, use `npx inngest-cli@0.14 dev`
- If using Node >= 18.16.0, download, unzip, and directly run the `inngest.exe` binary from the 0.14.6 release here: https://github.com/inngest/inngest/releases/tag/v0.14.6

This is not ideal at all! Working on getting those Windows builds sorted.

**Comment by bloomfieldj** [comment]
Thank you so much @khill-fbmc! Your solution worked for me with a small tweak since I'm using bash:

> If using Node >= 18.16.0, download, unzip, and directly run the inngest.exe binary from the 0.14.6 release here: https://github.com/inngest/inngest/releases/tag/v0.14.6

I extracted the executable to {{project root}}/node_modules/.bin/inngest.exe and then added the following command to its package.json :

```
 "inn-win": "bash -c \"./node_modules/.bin/inngest.exe dev\""
```

**Comment by jpwilliams** [comment]
This issue should now be resolved in [v0.16.4](https://github.com/inngest/inngest/releases/tag/v0.16.4) following #575.

Windows builds for >v0.14 are still being worked on in #521 and #550, so `npx inngest-cli@latest dev` on Windows will now use the latest supported version for that platform, [v0.14.6](https://github.com/inngest/inngest/releases/tag/v0.14.6).

**Comment by ronitrai27** [comment]
i have just fixed the issue for 
node version > 24.12
windows 11
using bun / pnpm / npm / yarn  or whatever --->

If u face error like Error: Inngest CLI binary not found.

so downlaod the file https://github.com/inngest/inngest/releases/download/v0.14.6/inngest_0.14.6_windows_x86_64.zip
then extract it to {root}/bin/ paste here your inngest.exe

then add a script -   "inngest:dev": "powershell -Command \"Start-Process '.\\bin\\inngest.exe' -ArgumentList 'dev' -NoNewWindow -Wait\"",

and run it like ---> bun run inngest:dev

this will 100% work !
